### ISSUE TITLE
What is the meaning of (N, T, C) ?

### ISSUE THREAD

**Issue by ken-take-it-so-so** [awslabs/gluon-ts#2340]
Hello.  
I read src/gluonts/mx/block/scaler.py and found below notation.  
https://github.com/awslabs/gluonts/blob/015ff06ae78ef78d7f5d73ccceef7ceb9d4db095/src/gluonts/mx/block/scaler.py#L32  
I want to know the meaning of each of N, T, and C. This expression often appears in the code. I think this problem is relevant to tensor shape.  
</br>
I would appreciate your cooperation.

**Comment by lostella** [comment]
@ken-take-it-so-so `N` = batch axis, `T` = time axis, `C` = channel axis; for univariate time series, usually the `C` axis is not there, so the tensor really has shape `(N, T)`

**Comment by ken-take-it-so-so** [comment]
@lostella, thank you for your response.  
I want to use multivariate time series. In this case, the interpretation of `C` = channel is just like dimension(e.g. the number of countries in exchange rate data) ? Is this right ?

**Comment by lostella** [comment]
@ken-take-it-so-so not really: `exchange_rate` is a dataset of univariate time series, see the following snippet

```python
>>> from gluonts.dataset.repository.datasets import get_dataset

>>> dataset = get_dataset("exchange_rate")

>>> next(iter(dataset.train))
{'start': Period('1990-01-01', 'B'),
 'target': array([0.7855  , 0.7818  , 0.7867  , ..., 1.023395, 1.023552, 1.025347],
       dtype=float32),
 'feat_static_cat': array([0], dtype=int32),
 'item_id': 0}
```

The `target` has only the time dimension.

**Comment by ken-take-it-so-so** [comment]
@lostella  
I want to use `gluonts.dataset.multivariate_grouper.MultivariateGrouper` and `gluonts.model.deepvar.DeepVAREstimator`.  

```python
from gluonts.dataset.repository.datasets import get_dataset
from gluonts.dataset.multivariate_grouper import MultivariateGrouper
from gluonts.model.deepvar import DeepVAREstimator
from gluonts.mx import Trainer

dataset = get_dataset("exchange_rate")
train_grouper = MultivariateGrouper(max_target_dim=min(2000, int(dataset.metadata.feat_static_cat[0].cardinality)))
dataset_train = train_grouper(dataset.train)
print(next(iter(dataset_train)))

estimator = DeepVAREstimator(
    target_dim=min(2000, int(dataset.metadata.feat_static_cat[0].cardinality)),
    prediction_length=dataset.metadata.prediction_length,
    context_length=dataset.metadata.prediction_length,
    freq=dataset.metadata.freq,
    num_layers=2,
    num_cells=40,
    scaling=True,
    batch_size=64,
    trainer=Trainer(
        epochs=5,
        num_batches_per_epoch=100,
        learning_rate=1e-3,
    )
)
```
Output
```
{'target': array([[0.7855  , 0.7818  , 0.7867  , ..., 1.023395, 1.023552, 1.025347],
       [1.611   , 1.61    , 1.6293  , ..., 1.607446, 1.6072  , 1.606813],
       [0.861698, 0.861104, 0.86103 , ..., 1.020721, 1.020721, 1.022066],
       ...,
       [0.006838, 0.006863, 0.006975, ..., 0.012752, 0.012753, 0.012697],
       [0.593   , 0.594   , 0.5973  , ..., 0.81686 , 0.81686 , 0.819001],
       [0.525486, 0.523972, 0.526316, ..., 0.81813 , 0.81813 , 0.818424]],
      dtype=float32), 'start': Period('1990-01-01', 'B'), 'feat_static_cat': array([0], dtype=int32)}
```

I want to discuss this case. Please bear with me.

**Comment by lostella** [comment]
Ah I see, ok, thanks for clarifying. Yes, in that case `C` would be the number of rows of `target`

**Comment by ken-take-it-so-so** [comment]
Thank you for teaching me.
